indico/modules/events/abstracts/templates/display/call_for_abstracts.html

```diff
@@ -123,11 +121,32 @@
         <section class="cfa-infoline-section">
             {{ render_cfa_infoline(event, cfa, show_icon=false) }}
         </section>
-        {% if abstracts %}
-            {{ render_user_abstract_list_section(event, abstracts, cfa) }}
-        {% endif %}
+        <section>
```

Comment:
Why not keep this inside the macro? This avoids reindenting everything and also makes more sense - right now you have an empty section if the user has no abstracts.